components/filter/filter.js

```diff
@@ -128,6 +128,14 @@ class Filter extends FocusMixin(LocalizeCoreElement(RtlMixin(LitElement))) {
 				line-height: unset;
 				overflow: hidden;
 			}
+			
+			.d2l-filter-dimension-set-value-text {
```

Comment:
Nit: note that the style linter will give errors for these properties as written. The properties need to be listed alphabetically, and there should be a space after the colon in each case (ex. `hyphens: auto;`).